core/src/components/picker-internal/picker-internal.md.scss

```diff
@@ -1,10 +1,13 @@
 @import "./picker-internal.scss";
 @import "../../themes/ionic.globals.md";
 
+$fallback: var(--background-rgb, var(--ion-background-color-rgb, 255, 255, 255));
+$background: var(--picker-fade-background-rgb, #{$fallback});
```

Comment:
Can we move these vars to a `picker-internal.vars.scss` file and have `picker-internal.{ios, md}.scss` pull from it?